### DISCUSSION TITLE
Web UI log into Tesla account times out...

### DISCUSSION THREAD

**Discussion by daviegeepers** [matt-FFFFFF/hassio-addon-repository#18] [General]
Hi there. I got most things installed properly. However, when I try to run teslamate from the homeassistant configuration screen I get the following error: 

[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[07:05:39] INFO: Checking for Grafana folder: TeslaMate
jq: error (at <stdin>:0): Cannot index string with string "title"
[cont-finish.d] executing container finish scripts...
s6-svscanctl: fatal: unable to control /var/run/s6/services: supervisor not listening
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.

**Comment by daviegeepers** [comment]
here is my teslamate config: 

database_user: teslamate_dave
database_pass: PASSWORD
database_name: teslamate_db
database_host: 29b65938-postgres
database_ssl: false
database_port: 5432
disable_mqtt: false
grafana_import_dashboards: true
grafana_folder_name: TeslaMate
grafana_host: a0d7b954-grafana
grafana_port: 3000
grafana_user: admin
grafana_pass: PASSWORD
import_dir: /share/teslamate
mqtt_host: core-mosquitto
mqtt_user: mqttuser_dave
mqtt_pass: PASSWORD
mqtt_tls: false
mqtt_tls_accept_invalid_certs: true
mqtt_namespace: ''
timezone: Europe/London

**Reply by matt-FFFFFF** [reply]
Hi,

Can you try and get it working with import_dashboards set to `false` - I think there might be a bug in the dashbaords.sh script.

**Reply by erlendsellie** [reply]
> Hi,
> 
> Can you try and get it working with import_dashboards set to `false` - I think there might be a bug in the dashbaords.sh script.

I'm struggling with the same issue. How would i go about fixing it? Do i need to manually import the dashboards?
Thanks for the addon!

**Comment by daviegeepers** [comment]
I’ll give it a shot. BRB.

**Comment by HA-TB303** [comment]
I have the same issue. Disabled grafana_import_dashboards and then teslamate wil start.

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[17:21:36] INFO: Checking for Grafana folder: TeslaMate
jq: error (at <stdin>:1): Cannot index string with string "title"
[cont-finish.d] executing container finish scripts...
s6-svscanctl: fatal: unable to control /var/run/s6/services: supervisor not listening
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
```
Teslamate config:

```
database_user: teslamate
database_pass: SECRET
database_name: teslamate
database_host: 29b65938-postgres
database_ssl: false
database_port: 5432
disable_mqtt: false
encryption_key: myencryptionkey
grafana_import_dashboards: false
grafana_folder_name: TeslaMate
grafana_host: a0d7b954-grafana
grafana_port: 3000
grafana_user: admin
grafana_pass: SECRET
import_dir: /share/teslamate
mqtt_host: 192.168.207.101
mqtt_user: teslamate
mqtt_pass: SECRET
mqtt_tls: false
mqtt_tls_accept_invalid_certs: false
mqtt_namespace: ""
timezone: Europe/Amsterdam
```

Grafana config:

```
plugins:
  - natel-discrete-panel
  - pr0ps-trackmap-panel
  - grafana-piechart-panel
env_vars:
  - name: GF_SECURITY_ADMIN_USER
    value: admin
  - name: GF_SECURITY_ADMIN_PASSWORD
    value: SECRET
ssl: true
certfile: fullchain.pem
keyfile: privkey.pem
```

**Comment by svedaker** [comment]
Same error for me: With grafana_import_dashboards set to false it works.

```[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[11:14:15] INFO: Checking for Grafana folder: TeslaMate
jq: error (at <stdin>:1): Cannot index string with string "title"
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
s6-svscanctl: fatal: unable to control /var/run/s6/services: supervisor not listening
[s6-finish] sending all processes the TERM signal.```

**Reply by horstvogel** [reply]
Same error; With grafana_import_dashboards set to false it works.

**Reply by yfguo** [reply]
This is because authentication faliure. The following Grafana configuration must be set before you start Grafana for the first time. Otherwise it won't work.
```
env_vars:
  - name: GF_SECURITY_ADMIN_USER
    value: admin
  - name: GF_SECURITY_ADMIN_PASSWORD
    value: mysecretpassword
```

If you following the steps in the tutorial where you setup postgres data source in Grafana before setting `env_vars`, you run into this problem.

You can simple remove the Grafana addon and reinstall it. Then update the configuration with both the `plugins` and `env_vars`. Then start Grafana for the first time and add data source.
